packages/uikit/src/components/Link/LinkExternal.tsx

```diff
@@ -3,16 +3,20 @@ import Link from "./Link";
 import { LinkProps } from "./types";
 import OpenNewIcon from "../Svg/Icons/OpenNew";
 import BscScanIcon from "../Svg/Icons/BscScan";
+import AptosScanIcon from "../Svg/Icons/AptosScan";
 
-const LinkExternal: React.FC<React.PropsWithChildren<LinkProps>> = ({ children, isBscScan = false, ...props }) => {
+const LinkExternal: React.FC<React.PropsWithChildren<LinkProps>> = ({
+  children,
+  isBscScan = false,
+  isAptosScan = false,
+  ...props
+}) => {
   return (
     <Link external {...props}>
       {children}
-      {isBscScan ? (
-        <BscScanIcon color={props.color ? props.color : "primary"} ml="4px" />
-      ) : (
-        <OpenNewIcon color={props.color ? props.color : "primary"} ml="4px" />
-      )}
+      {isBscScan && <BscScanIcon color={props.color ? props.color : "primary"} ml="4px" />}
+      {isAptosScan && <AptosScanIcon color={props.color ? props.color : "primary"} ml="4px" />}
```

Comment:
We ard have aptos logo icon
https://uikit.pancake.run/?path=/story/components-svg-icons--icons